ui/browser/views/tabbar/tabbar.jsx

```diff
@@ -40,7 +40,7 @@ const TabBar = ({ pageOrder, pages, currentPageIndex, dispatch, pageAreaVisible
           <Tab key={`browser-tab-${i}`} className={`browser-tab-${i}`}
             isActive={currentPageIndex === i}
             page={page}
-            onClick={() => dispatch(setCurrentTab(i))}
+            onClick={(e) => dispatch(clickTab(i, e.button))}
```

Comment:
Someone may need to correct me on arrow notation here, I can probably omit the parentheses.